controllers/constant/odlm.go

```diff
@@ -117,15 +117,6 @@ spec:
               spec:
                 imagePullSecrets:
                   - name: ibm-entitlement-key
-                affinity:
```

Comment:
```
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: kubernetes.io/arch
                operator: In
                values:
                - amd64
                - ppc64le
                - s390x
```

Could we update node affinity to this?

Reply:
@Daniel-Fan ^^^

Reply:
@horis233 Thank you for the reminder!

Updated~